iceoryx_posh/include/iceoryx_posh/popo/trigger_handle.hpp

```diff
@@ -37,7 +37,8 @@ namespace popo
 class TriggerHandle
 {
   public:
-    TriggerHandle() noexcept = default;
+    /// @warning do not use =default here otherwise QNX will fail to compile!
+    TriggerHandle() noexcept;
```

Comment:
We already had this once. I think removing `noexcept` helped that time but I'm not quite sure which class it was exactly.